extensions/resteasy-classic/resteasy-common/runtime/src/main/java/io/quarkus/resteasy/common/runtime/ResteasyCommonConfig.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.resteasy.common.runtime;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_AND_RUN_TIME_FIXED;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.quarkus.runtime.configuration.MemorySize;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigRoot(phase = BUILD_AND_RUN_TIME_FIXED)
+@ConfigMapping(prefix = "quarkus.resteasy")
+public interface ResteasyCommonConfig {
```

Comment:
I agree it's a cleaner approach. I'm not sure I understand exactly why things would fail specifically in native though?
I wouldn't expect the config to be less or more available?

Reply:
> I agree it's a cleaner approach. I'm not sure I understand exactly why things would fail specifically in native though?

because it is a build-time config and build-time config properties are not retained in a native mode, however it 
doesn't seem to be a case in JVM. maybe it's how our FW propagates properties. I can dig in if it is important...

> I wouldn't expect the config to be less or more available?

sorry if I avoid better answer because I am not sure what you mean, but we simply need this property available at runtime